torch/nn/modules/activation.py

```diff
@@ -432,9 +432,9 @@ class GLU(Module):
         dim (int): the dimension on which to split the input. Default: -1
 
     Shape:
-        - Input: :math:`(*, N, *)` where `*` means, any number of additional
+        - Input: :math:`(\ast_1, N, \ast_2)` where `*` means, any number of additional
```

Comment:
What's the difference between `\ast` and `*`?

Reply:
The code is inconsistent and flips between the two. I think \ast renders a bit nicer, particularly with subscripts.